pkgs/development/libraries/minc2-simple/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, cmake, libminc }:
+
+stdenv.mkDerivation {
+  version = "unstable-2018-12-20";
+  name    = "minc2-simple";
+
+  src = fetchFromGitHub {
+    owner  = "vfonov";
+    repo   = "minc2-simple";
+    rev    = "e04414eb74052d8a6f9fb4bedda7f0894e86ac82";
+    sha256 = "1miy942ibbcikg0k45d7342ags2gpr29rvhvspwxqgv2jnpcwg05";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ libminc ];
+
+  cmakeFlags = [ "-DBUILD_TESTING=OFF" "-DLIBMINC_DIR=${libminc}/lib/" ];
+
+  # no cmake 'install' target
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib $out/include
+    cp ../src/*.h $out/include
+    cp src/*.a src/*.so $out/lib
+
+    runHook postInstall
+  '';
+
+  # tests don't actually run automatically, e.g., missing input files
+  doCheck = false;
```

Comment:
A medical imaging library does seem like the sort of package we should at least make an effort to enable the tests for...

Reply:
I will work with upstream on this, but note that the tests for the Python interface to this library pass.